api/types/integration.go

```diff
@@ -34,6 +34,11 @@ const (
 	IntegrationSubKindAzureOIDC = "azure-oidc"
 )
 
+const (
```

Comment:
Can we also add the `discovery.service`?

Reply:
Hmm, I am not sure to add `discovery.service` right now, unless there is some code we know that will use the value right away. The `aws-identity-center` is coming right after this so it makes sense for that one but not for `discovery.service`.